modules/surface_matching/src/ppf_helpers.cpp

```diff
@@ -54,6 +54,20 @@ void getRandQuat(double q[4]);
 void getRandomRotation(double R[9]);
 void meanCovLocalPC(const float* pc, const int ws, const int point_count, double CovMat[3][3], double Mean[4]);
 void meanCovLocalPCInd(const float* pc, const int* Indices, const int ws, const int point_count, double CovMat[3][3], double Mean[4]);
+std::vector<std::string> split(const std::string &text, char sep);
```

Comment:
This forward declaration is not really doing anything here for us, since the definition comes right after it, you may remove it.